Prox/Prox/Data/PlacesController.swift

```diff
@@ -106,6 +107,7 @@ class PlacesProvider {
             failure: { (task, err) in
                 NSLog("Error from server: \(err)")
                 DispatchQueue.main.async {
+                    self.isUpdating = false
```

Comment:
When I was testing I saw that we weren't resetting the `isUpdating` flag on errors. Added a fix for that.